core/src/test/java/org/apache/calcite/util/graph/DirectedGraphTest.java

```diff
@@ -66,7 +66,7 @@
     assertNull(shortestPath(g, "A", "E"), "There is no path from A to E");
     assertEquals("[]", shortestPath(g, "D", "D").toString());
     assertNull(shortestPath(g, "X", "A"), "Node X is not in the graph");
-    assertEquals("[[A, B, C, D], [A, B, D]]", paths(g, "A", "D").toString());
+    assertEquals("[[A, B, D], [A, B, C, D]]", paths(g, "A", "D").toString());
```

Comment:
where are the tests for getShortestDistance()?

Reply:
We have DirectedGraphTest#testDistance